src/class-parsely.php

```diff
@@ -129,6 +131,53 @@ public function __construct() {
 		add_action( 'wp_enqueue_scripts', [ $this, 'wp_parsely_style_init' ] );
 	}
 
+	/**
+	 * Include a link to the statistics page for an article in the wp-admin Posts List.
+	 * If the post object is the "front page," this will include the main dashboard link instead.
+	 *
+	 * @see https://developer.wordpress.org/reference/hooks/page_row_actions/
+	 * @see https://developer.wordpress.org/reference/hooks/post_row_actions/
+	 * @param array<string, string> $actions The existing list of actions.
+	 * @param WP_Post               $post    The individual post object the actions apply to.
+	 * @return array<string, string> The amended list of actions.
+	 */
+	public function row_actions_add_parsely_link( $actions, $post ) {
+		if ( ! is_array( $actions ) ) {
+			return $actions;
+		}
+
+		if ( ! self::post_has_trackable_status( $post ) ) {
+			return $actions;
+		}
+
+		if ( ! self::post_has_viewable_type( $post ) ) {
+			return $actions;
+		}
+
+		$options = $this->get_options();
+		if ( ! (
+			isset( $options['apikey'] ) &&
+			is_string( $options['apikey'] ) &&
+			strlen( $options['apikey'] ) > 0
+		) ) {
+			return $actions;
+		}
+
+		$parsely_url = add_query_arg(
+			array(
+				'url' => rawurlencode( get_permalink( $post ) ),
+			),
+			trailingslashit( 'https://dash.parsely.com/' . $options['apikey'] ) . 'find'
+		);
+
+		$actions['find_in_parsely'] = sprintf(
+			'<a href="%1$s">%2$s</a>',
+			esc_url( $parsely_url ),
+			esc_html__( 'Parse.ly Stats', 'wp-parsely' )
+		);
```

Comment:
To have the same link text, pointing to different URLs, is a bad thing for accessibility. Links can be taken out of context, so they need to be more explainable.

Let's look at what WP core does for the existing Edit, Quick Edit, Trash, and View actions links:

<img width="1626" alt="Screenshot 2021-05-26 at 11 23 01" src="https://user-images.githubusercontent.com/88371/119644608-f4c35f00-be14-11eb-8a0f-57a020298288.png">

They use an `aria-label` attribute on the anchor element, that includes the (quoted) name of the post in some way. We should do the same for this Parse.ly Stats link.

Reply:
Addressed in c6032d9.